packages/apollo-voyager-tools/package.json

```diff
@@ -21,6 +21,7 @@
     "graphql-tools": "^4.0.3"
   },
   "devDependencies": {
+    "@aerogear/apollo-voyager-server": "0.2.7-rc3",
```

Comment:
This is circular dependency. No way this is going to work. 
We got lucky with the build.
Disabled that and logged separate issue #80

Reply:
@wtrocki this is only a dev dependency used in the tests to pull in some interface needed to allow compilation. It's not required in the production dependencies.

Reply:
Doesn't matter - when you will end up with clean project without stuff compiled typescript compiler will be forced to resolve `chicken or egg problem`.